apps/microtvm/reference-vm/base-box/base_box_provision.sh

```diff
@@ -20,12 +20,23 @@
 #   virtual machine similar to CI QEMU setup.
 #
 
-set -e
```

Comment:
because it is already added in first line

Reply:
it's not terrible to do this explicitly in the script actually--the first line is only ran when you directly invoke it as a command rather than just via `bash path/to/script.sh`